doc/src/build/custom-upgrade-policy.md

```diff
@@ -243,19 +195,91 @@ module policy::day_of_week {
 }
 ```
 
-We then need to add a function to authorize an upgrade, if we are on
-the correct day of the week:
+This code includes a constructor and defines the object type for the custom upgrade policy.
+
+You then need to add a function to authorize an upgrade, if on the correct day of the week. First, define a couple of constants, one for a day of the week where the upgrade isn't allowed, and another to define the number of milliseconds in a day (to be used shortly). Add these definitions directly under the current `ENotWeekDay` one.
```

Comment:
> one for a day of the week where the upgrade isn't allowed

`ENotAllowedDay` is the error code that gets emitted when the the policy is used on a day that isn't allowed, rather than being the not allowed day itself.

Reply:
Thanks! I figured that out by the time I got to the end but forgot about this line.